integration/localnet/Makefile

```diff
@@ -48,3 +48,19 @@ stop:
 .PHONY: load
 load:
 	go run --tags relic ../loader -log-level info -tps 1,0,10,0,100 -tps-durations 30s,10s,30s,10s
+
+unstaked-an:
```

Comment:
extra target to bring up a setup with 1 unstaked AN, 1 staked AN and the other nodes. This should help local testing.